### ISSUE TITLE
Feature request: Find text

### ISSUE THREAD

**Issue by Talkless** [fulvio999/tedit#20]
"Find text" functionality would be useful when opening bigger files. I use Tedit to summarize book I'm reading, but now I see that file is too big to find anything I want :) .

**Comment by fulvio999** [comment]
I agree that a "find" features is useful in case of big files (Tedit was made as a simple notes app for little notes).
I think is possible create it (at least searching in the currently opened note). The big problem is that i don't have time to develop it :(
